frontend/src/packages/sites/sites-api.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiPath, ContentType } from '~/libs/enums/enums.js';
+import { configureString } from '~/libs/helpers/helpers.js';
+import { HttpApi } from '~/libs/packages/api/api.js';
+import { type IHttp } from '~/libs/packages/http/http.js';
+import { type IStorage } from '~/libs/packages/storage/storage.js';
+import { SitesApiPath } from '~/packages/sites/libs/enums/enums.js';
+import { type SiteGetAllResponseDto } from '~/packages/sites/sites.js';
+
+type Constructor = {
+  baseUrl: string;
+  http: IHttp;
+  storage: IStorage;
+};
+
+class SitesApi extends HttpApi {
+  public constructor({ baseUrl, http, storage }: Constructor) {
+    super({ path: '', baseUrl, http, storage });
+  }
+
+  public async getByProjectId(
+    projectId: number,
+  ): Promise<SiteGetAllResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(
+        configureString(ApiPath.PROJECTS_$PROJECT_ID_SITES, SitesApiPath.ROOT, {
```

Comment:
```suggestion
        configureString(ApiPath.SITES, ProjectsApiPath.PROJECT_PROJECT_ID, {
```
the final path should be `/sites/project/:projectId`

Reply:
isn't resolved